Buildings/Experimental/DHC/Networks/Connections/Examples/Connection1PipeExample.mo

```diff
@@ -0,0 +1,150 @@
+  parameter Modelica.Units.SI.MassFlowRate mCon_flow_nominal=0.5
+    "Nominal mass flow rate in the connection line";
+  parameter Modelica.Units.SI.Length lDis=100
+    "Length of the distribution pipe before the connection";
+  Buildings.Experimental.DHC.Networks.Connections.Connection1PipeAutosize
+    connection1PipeAutosize(
+    redeclare package Medium = MediumW,
+    mDis_flow_nominal=mDis_flow_nominal,
+    mCon_flow_nominal=mCon_flow_nominal,
+    lDis=lDis,
+    dhDis=dh)
+    annotation (Placement(transformation(extent={{20,40},{40,60}})));
+  Buildings.Fluid.Sources.Boundary_pT bouDisAutoSize(
+    redeclare final package Medium = MediumW,
+    p=300000,
+    nPorts=1) annotation (Placement(transformation(extent={{-20,0},{0,20}})));
+  Modelica.Blocks.Sources.Constant mDis_flow_nominal_exp(k=mDis_flow_nominal)
+    annotation (Placement(transformation(extent={{-80,20},{-60,40}})));
+  Buildings.Experimental.DHC.EnergyTransferStations.BaseClasses.Pump_m_flow pumDisAutoSize(
+    dp_nominal=100000,
+    redeclare final package Medium = MediumW,
+    m_flow_nominal=mDis_flow_nominal) "Distribution network pump" annotation (
+      Placement(transformation(
+        extent={{-10,10},{10,-10}},
+        rotation=180,
+        origin={32,16})));
+
+  Buildings.Experimental.DHC.EnergyTransferStations.BaseClasses.Pump_m_flow pumConAutoSize(
+    dp_nominal=5000,
+    redeclare final package Medium = MediumW,
+    m_flow_nominal=mDis_flow_nominal) "Agent connection pump" annotation (
+      Placement(transformation(
+        extent={{10,10},{-10,-10}},
+        rotation=180,
+        origin={30,80})));
+  Buildings.Experimental.DHC.Networks.Connections.Connection1PipePlugFlow connection1PipeplugFlow(
+    redeclare package Medium = MediumW,
+    mDis_flow_nominal=mDis_flow_nominal,
+    mCon_flow_nominal=mCon_flow_nominal,
+    dIns=0.001,
+    kIns=1,
+    lDis=lDis)
+    annotation (Placement(transformation(extent={{20,-60},{40,-40}})));
+  Buildings.Fluid.Sources.Boundary_pT bouDisPlugFlow(
+    redeclare final package Medium = MediumW,
+    p=300000,
+    nPorts=1)
+    annotation (Placement(transformation(extent={{-20,-90},{0,-70}})));
+  Buildings.Experimental.DHC.EnergyTransferStations.BaseClasses.Pump_m_flow pumDisPlugFlow(
+    dp_nominal=100000,
+    redeclare final package Medium = MediumW,
+    m_flow_nominal=mDis_flow_nominal) "Distribution network pump" annotation (
+      Placement(transformation(
+        extent={{-10,10},{10,-10}},
+        rotation=180,
+        origin={32,-80})));
+  Buildings.Experimental.DHC.EnergyTransferStations.BaseClasses.Pump_m_flow pumConPlugFlow(
+    dp_nominal=5000,
+    redeclare final package Medium = MediumW,
+    m_flow_nominal=mDis_flow_nominal) "Agent connection pump" annotation (
+      Placement(transformation(
+        extent={{10,10},{-10,-10}},
+        rotation=180,
+        origin={30,-20})));
+  Fluid.FixedResistances.BuriedPipes.PipeGroundCoupling           pipeGroundCoupling(
```

Comment:
Please remove these long empty spaces

Reply:
I removed all the superfluous white space across the updated models